Fix '--where' experiment index filter on info

dapomeroy opened the pull request:
Fixes a bug where filtering using '--where' on the experiment index does not work correctly

dapomeroy committed:
Fix '--where' experiment index filter on info

Fixes a bug where filtering using '--where' on the experiment index does
not work correctly

linsword13 reviewed (commented):
Good stuff! Wonder if we can add a test for it?

dapomeroy committed:
Add test for workspace info with where filter

Adds a test to check for correct indexing using where filter in
workspace info.

douglasjacobsen commented on lib/ramble/ramble/cmd/workspace.py:
```diff
@@ -662,6 +662,18 @@ def workspace_info(args):
+                        app_inst.keywords.experiment_index
+                    )
+
+                    print_app_inst = print_experiment_set.get_experiment(exp_name)
```
The second argument in the loop is the same as the print_app_inst. So, I would suggest we change the loop to:

`for exp_name, print_app_inst, _ ...`

and then removing this line.

dapomeroy replied:
Created https://github.com/GoogleCloudPlatform/ramble/pull/644 to fix this

douglasjacobsen reviewed (changes requested):
LG One small request, but otherwise this is great!
